JulioC-19/LargeProjectMobile
# Julio/bottom tab bar

Implemented bottom tab bar. The condition for log in is hardcoded to true for the time being so it will direct you to the main screen with the bottom nav bar and the three screens. Once we have the api endpoints we'll hook up the log in and register screen again. 

To test: 
Run and build the app. 
It will take to home, navigate between the screen to make sure the bottom tab bar works.
